grails-app/views/plugin/Geneprint.gsp

```diff
@@ -19,7 +19,7 @@
                 <span>Select a High Dimensional Data node from the Data Set Explorer Tree and drag it into the box.</span>
                 <div id='divIndependentVariable' class="queryGroupIncludeSmall highDimBox"></div>
                 <div class="highDimBtns">
-                    <button type="button" onclick="highDimensionalData.gather_high_dimensional_data('divIndependentVariable')">High Dimensional Data</button>
+                    <button type="button" onclick="highDimensionalData.gather_high_dimensional_data('divIndependentVariable', false, false)">High Dimensional Data</button>
```

Comment:
There are two parameters being passed here, the first of which is false (or undefined) by default. The second is true by default, so only for geneprint it is explicitly set to false.